addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -323,7 +323,8 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
             )
         if len(self.channel_id.rtc_session_ids) == 1 and self.channel_id.channel_type != "channel":
             self.channel_id.message_post(body=_("%s started a live conference", self.partner_id.name or self.guest_id.name), message_type='notification')
-            self._rtc_invite_members()
+            if (not self.guest_id):
```

Comment:
parenthesis are not necessary

Aso I'm not sure but could a self have both a guest and a user? Like a user with a guest cookie on the public page? If so check if there is no partner